### ISSUE TITLE
API V2 and sandbox

### ISSUE THREAD

**Issue by DaleCollins** [robbiepaul/cloudconvert-laravel#64]
Hi,
First up, thanks for the Laravel wrapper.
I was wondering how you setup to use API v2 so i can use the sandbox, i can only seem to get this to work with API V1. The project is on Laravel 5.4
Is it possible? I'm burning through my minutes.

Edit: oops, just realised API V2 is still in preview. Never mind 

Thanks

**Comment by josiasmontag** [comment]
Correct, v2 is still in preview. We are working on this Laravel package for v2: https://github.com/cloudconvert/cloudconvert-laravel (WIP)

**Comment by robbiepaul** [comment]
@josiasmontag Nice! Let me know when it's ready and I'll sunset this package to point people to that one
